src/components/message/message.test.js

```diff
@@ -0,0 +1,135 @@
+      <Message open={isOpen} onDismiss={() => setIsOpen(false)} {...props}>
+        Some custom message
+      </Message>
+    </div>
+  );
+};
+
+context("Tests for Message component", () => {
+  describe("should check Message component properties", () => {
+    it.each([
+      ["info", "rgb(51, 91, 112)"],
+      ["error", "rgb(205, 56, 75)"],
+      ["success", "rgb(0, 138, 33)"],
+      ["warning", "rgb(239, 103, 0)"],
+    ])(
+      "should check %s variant for Message component",
+      (variant, backgroundColor) => {
+        CypressMountWithProviders(<MessageComponent variant={variant} />);
+        variantPreview().should(
+          "have.css",
+          "background-color",
+          backgroundColor
+        );
+      }
+    );
+
+    it.each(testData)(
+      "should check %s as children for Message component",
+      (stringValue) => {
+        CypressMountWithProviders(<Message>{stringValue}</Message>);
+        messageChildren().should("have.text", stringValue);
+      }
+    );
+
+    it("should set className for Message component", () => {
+      CypressMountWithProviders(
+        <MessageComponent className="message-cypress-classname" />
+      );
+      messagePreview().should("have.class", "message-cypress-classname");
+    });
+
+    it("should check custom id for Message component", () => {
+      CypressMountWithProviders(<MessageComponent id="FooBar" />);
+      messagePreview().should("have.id", "FooBar");
+    });
+
+    it.each([true, false])(
+      "should check when open is %s for Message component",
+      (boolVal) => {
+        CypressMountWithProviders(<MessageComponent open={boolVal} />);
+        if (boolVal === true) {
+          messagePreview().should("be.visible");
+        } else {
+          messagePreview().should("not.exist");
+        }
+      }
+    );
+
+    it.each(testData)(
+      "should check %s title for Message component",
+      (title) => {
+        CypressMountWithProviders(<MessageComponent title={title} />);
+        messageTitle().should("have.text", title);
+      }
+    );
+
+    it.each([
+      [true, "rgba(0, 0, 0, 0)"],
+      [false, "rgb(255, 255, 255)"],
+    ])(
+      "should check %s for transparent background for Message component",
+      (boolVal, backgroundColor) => {
+        CypressMountWithProviders(<MessageComponent transparent={boolVal} />);
+        messagePreview().should(
+          "have.css",
+          "background-color",
+          backgroundColor
+        );
+      }
+    );
+
+    it.each([true, false])(
+      "should check showCloseIcon when it's %s for Message component",
+      (boolVal) => {
+        CypressMountWithProviders(<MessageComponent showCloseIcon={boolVal} />);
+        if (boolVal === true) {
+          messageDismissIcon().should("be.visible");
+        } else {
+          messageDismissIcon().should("not.exist");
+        }
+      }
+    );
+
+    it("should check close button aria-label for Message component", () => {
```

Comment:
This test should use the string data type const to test with, similar to the children and title tests, as this is a custom aria label that can be set with any string.